server/run.py

```diff
@@ -65,16 +66,19 @@ def load_user(user_id):
     # add CORS
     @app.after_request
     def after_request(response):
-        response.headers.add("Access-Control-Allow-Origin", f"{FRONT_END_HOST}")
-        response.headers.add(
-            "Access-Control-Allow-Headers", "Content-Type,Authorization,Set-Cookie"
-        )
-        response.headers.add("Access-Control-Allow-Methods", "GET,PUT,POST,DELETE")
-        response.headers.add("Access-Control-Allow-Credentials", "true")
-        response.headers.add("Access-Control-Expose-Headers", "X-User-Id")
-        if current_user.is_authenticated:
-            response.headers.add("X-User-Id", f"{current_user.id}")
-        return response
+        allowed_origins = [f"{FRONT_END_HOST}", "https://frontend404.herokuapp.com", "https://website404.herokuapp.com", "https://backend-404.herokuapp.com"]
```

Comment:
Instead of hardcoded values, we should use the values from the remote node DB table.

Reply:
can you do that? I have no idea how the DB has been set up

Reply:
yeah sure

Reply:
Yeah gonna leave this for after the deadline, putting a TODO for now.